tools/harness-thci/OpenThread.py

```diff
@@ -559,7 +559,10 @@ def __startOpenThread(self):
         ]:
             self.__setRouterSelectionJitter(1)
         elif self.deviceRole in [Thread_Device_Role.BR_1, Thread_Device_Role.BR_2]:
-            self.IsBackboneRouter = True
+            if self.DeviceCapability == OT12BR_CAPBS:
+                self.IsBackboneRouter = True
+            else:
+                self.IsBackboneRouter = False
```

Comment:
Is a Thread 1.3 BR also a Backbone Router? 
I think at least multicast routing is required?

Reply:
I would think that as a backbone router, you would necessarily have a DUA prefix, right?

Reply:
The issue we encountered was that the OMR prefix conflicts with the DUA prefix. Since you can only have 1 OMR prefix, once the DUA prefix is configured and an ip address instantiated, you can't allocate an ip address on the OMR prefix. These changes were to address that issue. We saw this issue on the reference devices in tests DH_3_1 and DH_3_2. With these change, those test cases could pass against our soc DUT because the OMR prefix that the TH configured was allowed on the network. I suspect this change would cause those cases to fail once again.

Reply:
I guess DUA prefix configuration is only required for 1.2 certification. So, we may not want to add the prefix for 1.3 tests. 
Mayb we can simply check `DeviceCapability`?

Reply:
Made a change to address this @simonlingoogle .  Replaced `IsBackboneRouter` to instead check device capabilities.